x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/connectors/create_connector/start_step.tsx

```diff
@@ -100,6 +117,33 @@ export const StartStep: React.FC<StartStepProps> = ({
                     'xpack.enterpriseSearch.createConnector.startStep.euiFormRow.connectorNameLabel',
                     { defaultMessage: 'Connector name' }
                   )}
+                  helpText={
+                    <>
+                      <EuiText
+                        size="xs"
+                        css={css`
+                          line-break: anywhere;
+                        `}
+                        color="danger"
+                      >
+                        {formError()}
+                      </EuiText>
+                      <EuiText
+                        size="xs"
+                        css={css`
+                          line-break: anywhere;
+                        `}
+                      >
+                        {i18n.translate(
+                          'xpack.enterpriseSearch.startStep.namesShouldBeLowercaseTextLabel',
+                          {
+                            defaultMessage:
+                              'It should be lowercase and cannot contain spaces or special characters.',
```

Comment:
"It" is unclear. This should specify exactly what should be lowercase etc: "The connector name"

Reply:
Now we are using a more concise help text `The connector name should be lowercase and cannot contain spaces or special characters`